tests/pyccel/scripts/print_strings.py

```diff
@@ -19,7 +19,7 @@
     print('\a')
     print('\b')
     print('\f')
-    print('\r')
+    #print('\r')
```

Comment:
I think `'\r'` was working before.
can you uncomment this line to be checked again?

Reply:
Tests are passing d5b2f40a